frappe/patches/v14_0/set_hold_queue_default.py

```diff
@@ -0,0 +1,20 @@
+import frappe
+from frappe.cache_manager import clear_defaults_cache
+
+
+def execute():
+	frappe.db.set_default(
+		"hold_queue",
```

Comment:
If we're patching anyway, maybe we should rename this default to something more specific like `hold_email_queue`?